backend/controllers/contractorController.js

```diff
@@ -86,8 +87,9 @@ const updateContractorProfile = async (req, res) => {
     try {
         const contractorId = req.user._id;
         // Get the fields we want to update
-        const { description, phone_number, city, state, zip_code, website_url, services } = req.body;
+        const { description, phone_number, city, state, zip_code, website_url, services, profile_name } = req.body;
```

Comment:
The `profile_name` field can use the existing `name` field. There is no need to create a new field for this. Please replace the `profile_name` with name.

Reply:
profile_name and name are two different fields with name being the contractor's person's name and profile_name being the production name shown on listing